\documentclass[12pt]{article}
\usepackage{fullpage}
\usepackage{fancyhdr}

\usepackage{amsmath}
\usepackage{amssymb}

\usepackage{listings}
\lstset{language=Python,
        basicstyle=\footnotesize\ttfamily,
        showspaces=false,
        showstringspaces=false,
        tabsize=2,
        breaklines=false,
        breakatwhitespace=true,
        identifierstyle=\ttfamily,
        keywordstyle=\bfseries,
        commentstyle=\it,
        stringstyle=\it,
    }

\usepackage[pdftex]{graphicx}

% header
\fancyhead{}
\fancyfoot{}
\fancyfoot[C]{\thepage}
\fancyhead[R]{Daniel Foreman-Mackey}
\fancyhead[L]{Biophysics --- Problem Set 3}
\pagestyle{fancy}
\setlength{\headsep}{25pt}

% shortcuts
\newcommand{\Eq}[1]{Equation (\ref{eq:#1})}
\newcommand{\eq}[1]{Equation (\ref{eq:#1})}
\newcommand{\eqlabel}[1]{\label{eq:#1}}
\newcommand{\Fig}[1]{Figure \ref{fig:#1}}
\newcommand{\fig}[1]{Figure \ref{fig:#1}}
\newcommand{\figlabel}[1]{\label{fig:#1}}

% commands
\newcommand{\bvec}[1]{\ensuremath{\boldsymbol{#1}}}
\newcommand{\unit}[1]{\ensuremath{\,\mathrm{#1}}}

\begin{document}

% === Problem 1 ===
\section{Problem 1 --- FRAP}

\paragraph{(a)}

In this model, starting from an initial concentration $c_0$ at time $t=0$, the
time evolution of $c(t)$ will be
\begin{eqnarray}
c(t) &=& \frac{k_\mathrm{on}}{k_\mathrm{off}}
    + \left[ c_0 - \frac{k_\mathrm{on}}{k_\mathrm{off}} \right] \,
    \exp \left( -k_\mathrm{off}\,t \right) \quad.
\end{eqnarray}
In steady state,
\begin{eqnarray}
\frac{dc}{dt} = 0 &\rightarrow& c = \frac{k_\mathrm{on}}{k_\mathrm{off}}
\end{eqnarray}
and if we FRAP a fraction $\alpha$ of the molecules (and then start the
clock), we get
\begin{eqnarray}
c_0 &=& (1-\alpha)\,\frac{k_\mathrm{on}}{k_\mathrm{off}}
\end{eqnarray}
giving
\begin{eqnarray}
c(t) &=& \frac{k_\mathrm{on}}{k_\mathrm{off}} \,
    \left [
        1 - \alpha\,\exp \left( -k_\mathrm{off}\,t \right)
    \right ] \quad.
\end{eqnarray}

\paragraph{(b)}

Using this model, we can't constrain the value of the binding rate
$k_\mathrm{on}$ because it is exactly degenerate with the response of the
detector.
To fit for the exact value of $k_\mathrm{off}$, we would also need to know (or
simultaneously fit for) the value of $\alpha$ because, while it isn't exactly
degenerate, I think that you'd need to collect some pretty great data if you
want to infer both parameters.
Either way, we can clearly say that \emph{S5} has the smallest value of
$k_\mathrm{off}$, and \emph{UBF} and \emph{Nucleolin} have the largest value.
Assuming $\alpha \sim 1$, (and using a ruler on the Figure)
\begin{itemize}
{\item for \emph{UBF} and \emph{Nucleolin}, $k_\mathrm{off} \approx 0.08$,}
{\item for \emph{B23} and \emph{Rpp}, $k_\mathrm{off} \approx 0.04$, and}
{\item for \emph{S5}, $k_\mathrm{off} \approx 0.006$.}
\end{itemize}



% === Problem 2 ===
\section{Problem 2 --- Diffusion limited aggregation}

\paragraph{(a)}

In his 1997 paper, Odde \cite{odde} derived an analytic model for the
diffusion-limited growth of microtubules and I will follow this derivation.
We'll start with the steady-state diffusion equation in cylindrical
coordinates aligned with the relevant cytoskeletal filament but with the
origin defined at the growing tip
\begin{eqnarray}
0 &=& D\,\left[
\frac{1}{r}\,\frac{d}{dr} \left(r\,\frac{dc}{dr}\right) + \frac{d^2c}{dz^2}
\right] - v\,\frac{dc}{dz}
\end{eqnarray}
where $c(r,\,z)$ is the concentration profile and $v$ is the rate of growth at
the tip.
Then, using the radial distance $s^2=r^2+s^2$, we can define the relevant
boundary conditions:
\begin{itemize}
\item at large radius, the concentration should become equal to the bulk
concentration, $c\to c_\infty$ when $s\to\infty$,
\item at the surface of the tip ($s = R$), the flux will
be equal to some (yet unknown) constant $W = 4\,\pi\,s^2\,D\,(dc/ds)$, and
\item the concentration profile should be cylindrically symmetric $dc/dr=0$
at $r=0$.
\end{itemize}
The solution to this system is given in Equation (4) in \cite{odde} as
\begin{eqnarray}\eqlabel{odde}
c(r,\,z) &=& c_\infty - \frac{w}{4\,\pi\,D\,s}\,\exp\left( -\frac{v}{2\,D}
    \left[ s-z \right] \right)
\end{eqnarray}
where $w = K\,v$ is the rate of growth in units of monomer per time, and $K$
will be a different constant for each polymer.

Then, Odde argues that the term $v / D$ will, in practice, be very small
compared to $s \ge R$ so \eq{odde} will be dominated by the $1/s$ term and the
exponential term can be neglected giving
\begin{eqnarray}
c(s) &=& c_\infty - \frac{w}{4\,\pi\,D\,s} \quad.
\end{eqnarray}
It's obvious from this equation that the growth rate $v \propto w$ will depend
on the details of the concentration gradient but the maximum possible growth
rate occurs when $c(R) = 0$, or
\begin{eqnarray}
w_\mathrm{max} &=& 4\,\pi\,D\,R\,c_\infty \quad.
\end{eqnarray}

For microtubules, using the values, $D \sim 5\,\mathrm{\mu m^2/s}$,
$R \sim 12.5\,\mathrm{nm}$, and $c_\infty \sim 10\,\mathrm{\mu M}$ this works
out to give
\begin{eqnarray}
w_\mathrm{max} &\sim& 8 \times 10^{-15} \, \mathrm{\mu mol\,s^{-1}} \quad.
\end{eqnarray}
Using the fact from problem set 1 that the density of tubulin dimers in a
microtubule is $1690\,\mathrm{\mu m^{-1}}$, we can compute the conversion from
$w$ to $v$
\begin{eqnarray}
v &=& \frac{w}{4.67\times10^{-17} \, \mathrm{min\,\mu mol\,s^{-1}\,\mu
        m^{-1}}} \\
\to \quad v_\mathrm{max} &=& 168 \, \mathrm{\mu m \, min^{-1}} \quad.
\end{eqnarray}

Similarly, for actin, the subunit density is $370\,\mathrm{\mu m^{-1}}$
\cite{pollard}, $R\sim 4\,\mathrm{nm}$, and $c_\infty \sim 20\,\mathrm{\mu
M}$ giving
\begin{eqnarray}
v_\mathrm{max} &=& \frac{5\times10^{-15} \, \mathrm{\mu mol\, s^{-1}}}
            {10^{-17} \, \mathrm{min\,\mu mol\,s^{-1}\,\mu m^{-1}}} \\
&=& 490 \, \mathrm{\mu m \, min} \quad.
\end{eqnarray}

\paragraph{(b)}

\Fig{concent} shows the concentration profiles for these two polymers as
computed using \eq{odde} and the values from above.

\begin{figure}[htbp]
\begin{center}
\includegraphics[width=0.8\textwidth]{c.pdf}
\end{center}
\caption{%
The concentration profile of monomers as a function of the distance from the
growing polymer tip. The solid line shows the profile for a microtubule and
the dashed line shows the result for actin.
\figlabel{concent}}
\end{figure}

\paragraph{(c)}

In agreement with the results in \cite{odde}, this predicted growth rate is
far greater than the measured value.
The conclusion from the referenced paper is that this means that the growth
cannot possibly be diffusion limited.

\paragraph{(d)}

From the half-time for recovery, we can use the simple model from problem 1 to
estimate the \emph{unbinding} rate $k_\mathrm{off} = \ln 2 / \tau_{1/2}$.
In the bundle, this gives $k_\mathrm{off} \sim 3\times 10^{-3} \,\mathrm{Hz}$.
Assuming that $k_\mathrm{on} \approx k_\mathrm{off}$, that the bleached
volume was $\sim (10\,\mathrm{\mu m})^3$, and that the density of actin
monomers is $20\,\mathrm{\mu M}$, I find that the growth rate is
\begin{eqnarray}
w &\approx& 7 \times 10^{-17} \, \mathrm{\mu mol \, s^{-1}} \\
v &\approx& 6.8 \, \mathrm{\mu m\,min^{-1}} \quad.
\end{eqnarray}
For the actin at the edge of the cell, this works out to be
\begin{eqnarray}
w &\approx& 7 \times 10^{-16} \, \mathrm{\mu mol \, s^{-1}} \\
v &\approx& 68 \, \mathrm{\mu m\,min^{-1}} \quad.
\end{eqnarray}
These two results are again much lower than the result estimated in the
diffusion limited case but it all seems a bit hand-wavy to me and changing any
of the assumptions makes a huge difference in the result.


% === Problem 3 ===
\section{Problem 3 --- Tubulin purification}

The drawings for the problem are shown on the last page.

\paragraph{(a)}

In step (1), the blender is being used to break up the cells, bursting the
membranes but probably keeping most of the organelles intact.
In step (2) the cytoplasm (including the microtubules) is separated from the
other solids using a centrifuge.
Then in step (3) the cytoplasm if mixed with some ATP and GTP to fuel
polymerization in this solution.
This will cause the microtubules (and some other polymers) to polymerize into
more massive compounds.

\paragraph{(b)}

The polymerized solution is centrifuged in step (4) to separate the polymers
(the solid ``pellet'') from the remaining components of the cytoplasm (the
liquid or ``supernatant'').
In step (5), the unwanted cytoplasm is discarded and the pellet (where the
polymerized tubulin and some other polymers now live) is resuspended and
cooled to depolymerize.

\paragraph{(c)}

In steps (6) and (7) this depolymerized solution is again centrifuged to
separate the depolymerized tubulin from (I expect) some remaining other
polymers that aren't depolymerized as quickly.
The supernatant isn't discarded because the tubulin monomers will be dissolved
in it at this point.
Then, also in step (7), we add some more fuel and re-polymerize the tubulin.
This time, the process is run for less time and at a colder temperature
meaning that some polymers that require more time or warmer conditions will
remain depolymerized.

\paragraph{(d)}

Then this solution is centrifuged (in step 8) to separate the (now
polymerized) tubulin from the remaining solution and then the resulting solid
is collected for the following step.
In step (9) this pellet is again resuspended and depolymerized.

\paragraph{(e)}

After these two rounds of polymerization/depolymerization, the tubulin is
almost ready to go but I guess there are still a few polymers that couldn't be
completely removed.
The phosphocellulose column is used to remove DNA binding proteins from a
solution \cite{pc}.


% === Problem 4 ===
\section{Problem 4 --- Nobel prize}

I didn't really know where to start with this problem but in searching on the
internet I came across a set of predictions by researchers from Thomson
Reuters \cite{predict}.
To generate their prediction, they (algorithmically) study a proprietary
scientific citation network that they have built and sometimes it gives good
predictions but they've only correctly predicted 35 of the Nobel winners in
the past 12 years.
Even so, this seemed like a good place to start so I'll describe the three
results that they consider Nobel worthy.

The first discovery that they predict is related to \emph{eukaryotic
transcription} and they predict that the Nobel would be awarded to James E.
Darnell Jr., Robert G. Roeder, and Robert Tjian.
Eukaryotic transcription is the process that converts the information in DNA
to RNA for translation \cite{eukaryotic}.
These researchers are responsible for discovering proteins for regulating gene
expression.
In particular, Roeder was awarded the Lasker Award in 2003 \cite{lasker} for
his lab's work on determining the proteins necessary for specific gene
activations in mammals with the ultimate goal of treating serious diseases
genetically.

The second prediction goes to David Julius for his research into the molecular
basis for the sensation of pain (or nociception \cite{nociception}).
Julius received the Shaw Prize in 2010 for work done in his lab studying
specific ion channels related to pain related messaging.
His most famous paper was published in Nature in 1997 \cite{julius} where they
extracted capsaicin (the chemical responsible for the ``burning'' sensations)
from hot peppers and studied its interaction with cells \emph{in vitro}.

The last Nobel prediction that I'll look at here is for research into
\emph{DNA copy-number variation}, a process where a certain gene or set of
genes end up with abnormal numbers of copies in the genome \cite{cnv}.
These mutations of the genome have been linked to diseases and hereditary
conditions both harmful and beneficial.
In particular, copy-number variations have been linked (by researchers in Dr.
Wigler's lab) to autism \cite{novo}.
The researchers at Thomson Reuters predict that this Nobel would go to Charles
Lee, Stephen W. Scherer, and Michael H. Wigler.
Each of these researchers lead a lab where specific copy-number variations
were discovered and characterized.

\newpage
\mbox{}
\newpage


% === Bibliography ===
\bibliography{ps3}{}
\bibliographystyle{unsrt}

\end{document}
